packages/cambodia/cambodia.svg

```diff
@@ -0,0 +1,109 @@
+<svg
+   xmlns="http://www.w3.org/2000/svg"
+   viewBox="0 0 655 655"
+   aria-label="Map of Cambodia"
+>
+  <path
+     id="banteay-meanchey"
```

Comment:
Could you use the same English names as in the README for `id` and `name` please?

Reply:
Yes, I can.
Here is example for what I was changed in the README, id and name on Cambodia.svg

> In README
- Kampong Chhnang
> In cambodia.svg
- id: kampong-chhnang
- name: Kampong Chhnang

give me a feedback please. If that have some issue.
thanks in advance 🙂

Reply:
That's perfect 👌